src/main/java/seedu/address/logic/parser/FindEventCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.event.EventContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindEventCommand object
+ */
+public class FindEventCommandParser implements Parser<FindEventCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindEventCommand
+     * and returns a FindEventCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format.
+     */
+    public FindEventCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
```

Comment:
Great use of trim to ensure no errors are thrown in case user types in an extra space